src/components/display/LoadingBody.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react"
```

Comment:
I think we should rename this component to `SkeletonTableBody`, I think it makes more sense in the context of a standalone library